src/app/geo-header/geo-header.component.ts

```diff
@@ -28,4 +28,23 @@ export class GeoHeaderComponent {
       .dispatch(changeSearchFromAddress($value));
   }
 
+  geolocate() {
+    this.store
```

Comment:
Geolocation has no change state, for me is an external service that gives coordenates to make a possible change of geocoding state. We'll talk about it at the meeting if you want.